plugins/osdn/common.go

```diff
@@ -26,11 +26,11 @@ type PluginHooks interface {
 
 	SetupSDN(localSubnetCIDR, clusterNetworkCIDR, serviceNetworkCIDR string, mtu uint) (bool, error)
 
-	AddOFRules(nodeIP, nodeSubnetCIDR, localIP string) error
-	DelOFRules(nodeIP, localIP string) error
+	HostSubnetAdded(subnet *osapi.HostSubnet)
+	HostSubnetDeleted(subnet *osapi.HostSubnet)
 
-	AddServiceOFRules(netID uint, IP string, protocol kapi.Protocol, port int) error
-	DelServiceOFRules(netID uint, IP string, protocol kapi.Protocol, port int) error
+	ServiceAdded(service *kapi.Service, netID uint)
+	ServiceDeleted(service *kapi.Service, netID uint)
 
```

Comment:
The intent of the function isn't "add a HostSubnet" though. It's called after the HostSubnet has already been added, so the intent-based name would be "UpdateStateForAddedHostSubnet" or something.

(The old names were intent-based, but I was trying to switch to names that didn't presuppose the use of OpenFlow rules...)